### PR TITLE
chore: Update ubuntu to 22.04 and actions to node 16 versions

### PR DESCRIPTION
**Pull request by SDawley**
Signed-off-by: sdawley <sdawley@redhat.com>

### What does this PR do?
Updates Ubuntu to 22.04 and all actions to the latest version using node 16

### What issues does this PR fix?

https://github.com/eclipse/che/issues/21763

### CODE REVIEW

File: .github/workflows/try-in-web-ide.yaml [hunk 1]
```diff
@@ -18,16 +18,16 @@ on:
...
       - name: Add DevSandbox link
-        uses: redhat-actions/try-in-web-ide@v1
+        uses: redhat-actions/try-in-web-ide@v1.2
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           add_comment: true
           add_status: false
       - name: Add Dogfooding link
-        uses: redhat-actions/try-in-web-ide@v1
+        uses: redhat-actions/try-in-web-ide@v1.2
```
Comment (nickboldt) [inline .github/workflows/try-in-web-ide.yaml:30]: why 1.2 here? are we avoiding 1.3+ ?

Comment (azatsarynnyy) [reply .github/workflows/try-in-web-ide.yaml:30]: Currently, 1.2 is the latest - https://github.com/redhat-actions/try-in-web-ide/releases

**Review by azatsarynnyy — APPROVED** [review]
Thanks @SDawley for checking all the actions!

File: .github/workflows/try-in-web-ide.yaml (hunk 1 above)
Comment (azatsarynnyy) [reply .github/workflows/try-in-web-ide.yaml:30]: Currently, 1.2 is the latest - https://github.com/redhat-actions/try-in-web-ide/releases

Comment (nickboldt) [reply .github/workflows/try-in-web-ide.yaml:30]: yes, but @1 == @1.x so this change is actually PINNING to 1.2 rather than using 1.latest

**Pull request merged**